lang/operator.md

```diff
@@ -126,7 +126,7 @@ impl<M: Memory> Machine<M> {
         // `offset.abs()` will fit into a `Size` since we did the overflow check above.
         // FIXME: actually, it could be isize::MIN and then everything breaks? Is that
         // a valid offset?
-        self.mem.dereferenceable(min_ptr, Size::from_bytes(offset.abs()), Align::ONE)?;
+        self.mem.dereferenceable(min_ptr, Size::from_bytes(offset.abs()), Align::from_bytes(1))?;
```

Comment:
There is no `const fn` BigInt constructor, as it's heap-allocated.
Another option would be `Align::one()`.

Reply:
Which type are you using for BigInt?
In principle a non-heap case would be possible for small numbers. But I can also live with `Align::one()`.

Reply:
I'm currently using the crate `num-bigint`.
Yeah I also thought whether we made some kind of
`enum Int { Big(BigInt), Small(i64) }`, would be kinda overkill though I think ^^